test/system/070-build.bats

```diff
@@ -210,6 +210,41 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman parallel build should not race" {
+    rand_filename=$(random_string 20)
+    rand_content=$(random_string 50)
+    #iterator
+    iter=1
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $tmpdir
+    mkdir -p $PODMAN_TMPDIR/reldir
+
+    containerfile=$PODMAN_TMPDIR/reldir/Containerfile
+    cat >$containerfile <<EOF
+FROM quay.io/libpod/alpine:latest
+RUN echo $rand_content > /$rand_filename
+EOF
+
+    run_podman rm --all --force --ignore
+    run_podman rmi -af
+    while [ $iter -le 30 ]
+    do
+      run_podman build -t $iter -f $containerfile &> /dev/null
+      podman_pid="$!"
+      iter=$(( $iter + 1 ))
+    done
+
+    # wait for last triggered build to complete
+    wait $podman_pid
```

Comment:
doesn't it work if you don't specify anything?

Waiting just  for the last pid is not correct, you don't know it will be the last one to complete

Reply:
ah yes it works without waiting for a specific $PID as well. Fixed in latest commit.